src/masternodeman.cpp

```diff
@@ -1609,22 +1606,17 @@ void CMasternodeMan::UpdateLastPaid(const CBlockIndex* pindex)
     IsFirstRun = false;
 }
 
-void CMasternodeMan::UpdateWatchdogVoteTime(const COutPoint& outpoint, uint64_t nVoteTime)
+void CMasternodeMan::UpdateLastSentinelPingTime()
 {
     LOCK(cs);
-    CMasternode* pmn = Find(outpoint);
-    if(!pmn) {
-        return;
-    }
-    pmn->UpdateWatchdogVoteTime(nVoteTime);
-    nLastWatchdogVoteTime = GetTime();
+    nLastSentinelPingTime = GetTime();
 }
 
-bool CMasternodeMan::IsWatchdogActive()
+bool CMasternodeMan::IsSentinelPingActive()
 {
     LOCK(cs);
     // Check if any masternodes have voted recently, otherwise return false
-    return (GetTime() - nLastWatchdogVoteTime) <= MASTERNODE_WATCHDOG_MAX_SECONDS;
+    return (GetTime() - nLastSentinelPingTime <= MASTERNODE_SENTINEL_PING_MAX_SECONDS);
```

Comment:
Hmm.... I guess I still don't see how the parens help with this placement. Wouldn't it be:

`(GetTime() - nLastSentinelPingTime) <= MASTERNODE_SENTINEL_PING_MAX_SECONDS`

... in order to first calculate `GetTime() - nLastSentinelPingTime` before the `<=` comparison?

Reply:
ok, I see that the fix is causing more confusion than it was meant to fix, let me just revert this then :)

Reply:
I probably could have been more descriptive and less nit-picky. :/ Thanks Udjin, looks good. 👍

Reply:
No, probs :) Thanks for the review! 👍